shell/platform/android/io/flutter/embedding/engine/systemchannels/LifecycleChannel.java

```diff
@@ -15,23 +16,69 @@
  */
 public class LifecycleChannel {
 
+  public enum AppLifecycleState { INACTIVE, RESUMED, PAUSED }
```

Comment:
Would you mind moving all enums and inner classes to the bottom of the file?

Also, please add comments for each enum value that describe what exactly they represent in terms of lifecycle.